site/en/blog/colrv1-fonts/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: COLRv1 Color Gradient Vector Fonts in Chrome 98
+description: >
+  In Chrome 98, the Chrome and Fonts teams have added support for COLRv1, an
+  evolution of the COLRv0 font format intended to make color fonts widespread by
+  adding gradients, compositing and blending, and improved internal shape reuse
+  for crisp and compact font files that compress well.
+layout:
+  'layouts/blog-post.njk'
+date: 2022-01-06
+authors:
+  - drott
+  - rsheeter
+hero: 'image/roKb5a4ddjOSoIbxRFI9kDeuVvE2/NQGoL7CmejHN0lUntQfz.png'
+alt: >
+  A sharp vs. a slightly blurry national park emoji and a bar chart highlighting
+  the size difference between COLRv1 and a bitmap font: about 9MB vs. 1.85MB.
+tags:
+  - chrome-98
+  - fonts
+---
+
+*Compact, compression-friendly, color vector fonts with all your favorite gradient flavors*
+
+In Chrome 98, the Chrome and Fonts teams have added support for COLRv1, an
+evolution of the COLRv0 font format intended to make color fonts widespread by
+adding gradients, compositing and blending, and improved internal shape reuse
+for crisp and compact font files that compress well.
+
+{% Aside 'caution' %}
+Examples in this blog post require a browser that supports
+COLRv1 fonts. Try viewing this post in Chrome 98 or above.
+{% endAside %}
+
+## Color Now
```

Comment:
```suggestion
## Color now
```

Reply:
Google web sites use sentence case for titles. I've never heard a reason. I'm guessing it simplifies machine translation since every alphabet based language has different rules for titles.

Reply:
Ack.